cpp/cmake/thirdparty/get_libcudacxx.cmake

```diff
@@ -16,7 +16,9 @@
 function(find_and_configure_libcudacxx)
   include(${rapids-cmake-dir}/cpm/libcudacxx.cmake)
 
-  rapids_cpm_libcudacxx(BUILD_EXPORT_SET raft-exports INSTALL_EXPORT_SET raft-exports)
+  rapids_cpm_libcudacxx(BUILD_EXPORT_SET raft-backend-exports
```

Comment:
We should simplify all of these `get_*` changes to the following instead of using `EXCLUDE_FROM_ALL`

```cmake
rapids_cpm_libcudacxx(BUILD_EXPORT_SET raft-backend-exports)
rapids_export_package(INSTALL libcudacxx raft-backend-exports)
```